scripts/vim-patch.sh

```diff
@@ -226,7 +226,7 @@ stage_patch() {
       printf "\n✘ 'patch' command not found\n"
     else
       printf "\nApplying patch...\n"
-      patch -p1 < "${patch_file}"
+      patch -p1 --no-backup-if-mismatch < "${patch_file}"
```

Comment:
`--no-backup-if-mismatch` is a GNU-ism.  `--posix` causes the same behavior, but also has some other effects.